docs/contributing/README.md

```diff
@@ -45,22 +47,29 @@ mkdocs serve
 
 ## Testing
 
-```bash
-pip install -r requirements/dev.txt
+=== "NVIDIA CUDA"
 
-# Linting, formatting and static type checking
-pre-commit install --hook-type pre-commit --hook-type commit-msg
+    ```bash
+    sudo apt install python3-dev
 
-# You can manually run pre-commit with
-pre-commit run --all-files
+    # work around for https://github.com/state-spaces/mamba/issues/720 that's also done in docker/Dockerfile
+    uv pip install --no-build-isolation "git+https://github.com/state-spaces/mamba@v2.2.4"
 
-# To manually run something from CI that does not run
-# locally by default, you can run:
-pre-commit run mypy-3.9 --hook-stage manual --all-files
+    uv pip install -r requirements/dev.txt --torch-backend=auto
 
-# Unit tests
-pytest tests/
-```
+    # Linting, formatting and static type checking
+    pre-commit install --hook-type pre-commit --hook-type commit-msg
```

Comment:
These extra args are no longer needed since https://github.com/vllm-project/vllm/blob/6e4cea1cc56da6d4a558a50196c5698c36385890/.pre-commit-config.yaml#L1-L3 was added

```suggestion
    pre-commit install
```